src/main/java/seedu/address/ui/UiManager.java

```diff
@@ -51,7 +52,7 @@ public void start(Stage primaryStage) {
     }
 
     private Image getImage(String imagePath) {
-        return new Image(MainApp.class.getResourceAsStream(imagePath));
+        return new Image(Objects.requireNonNull(MainApp.class.getResourceAsStream(imagePath)));
```

Comment:
Very nice 👍  but to follow the rest of the code, requireNonNull() can be imported.